src/infrastructure/db/repositories/decisions.repository.ts

```diff
@@ -41,6 +45,21 @@ export class DecisionsRepository implements InterfaceDecisionsRepository {
     return decision
   }
 
+  async removeById(id: string): Promise<void> {
+    const removalResponse = await this.decisionModel
+      .deleteOne({ _id: id })
+      .lean()
+      .catch((error) => {
+        throw new DatabaseError(error)
+      })
+    if (removalResponse.deletedCount === 0) {
+      throw new DecisionNotFoundError()
+    }
+    if (!removalResponse.acknowledged) {
```

Comment:
cas non testé dans le TU

Reply:
Cas rajouté en se basant sur acknowledged to false